### ISSUE TITLE
Demote x86_64-rumprun-netbsd target

### ISSUE THREAD

**Issue by nikic** [rust-lang/rust#81514]
The `x86_64-rumprun-netbsd` is currently a Tier 2 std target built on dist-various-1. In the course of updating that image, I found that the rumprun project appears to be unmaintained. There has been little activity recently (https://github.com/rumpkernel/rumprun) and there are outstanding PRs to fix build issues. The http://www.rumpkernel.org/ domain is down.

It *is* still possible to build rumprun on newer Ubuntu versions with the following rather minor adjustment:
```
diff --git a/src/ci/docker/host-x86_64/dist-various-1/build-rumprun.sh b/src/ci/docker/host-x86_64/dist-various-1/build-rumprun.sh
index 103dbbe6fda..bee2d7a476c 100755
--- a/src/ci/docker/host-x86_64/dist-various-1/build-rumprun.sh
+++ b/src/ci/docker/host-x86_64/dist-various-1/build-rumprun.sh
@@ -20,9 +20,10 @@ exit 1
 
 git clone https://github.com/rumpkernel/rumprun
 cd rumprun
-git reset --hard 39a97f37a85e44c69b662f6b97b688fbe892603b
+git reset --hard b04d42225a12a6fae57a78a9c1cf23642e46cd00
 git submodule update --init
 
-CC=cc hide_output ./build-rr.sh -d /usr/local hw
+# Disable -Werror, to avoid breaking the build with newer compilers.
+CC=cc NOGCCERROR=1 hide_output ./build-rr.sh -d /usr/local hw
 cd ..
 rm -rf rumprun
```

However, I question the utility of keeping an unmaintained project as a Tier 2 target.

**Comment by nagisa** [comment]
Nominating for T-compiler discussion. I suspect that the team will be comfortable with the demotion unless somebody expresses a justified reason why the demotion should not be demoted.

**Comment by apiraino** [comment]
agreed to demote and drop support during [compiler team meeting](https://zulip-archive.rust-lang.org/238009tcompilermeetings/89477weeklymeeting2021020454818.html#225178063)

**Comment by nagisa** [comment]
We discussed this during the @rust-lang/compiler meeting today.

This FCP is for the removal of this target from rustc. We felt that this target is somewhat similar to the Windows XP situation, support for which was removed recently as part of [this MCP](https://github.com/rust-lang/compiler-team/issues/378) in that the target is unmaintained for long enough that removing it is well warranted.

@rfcbot fcp merge
